torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -182,16 +180,11 @@ struct KinetoThreadLocalState : public ProfilerThreadLocalState {
             std::to_string(kineto_event.sequenceNr()));
       }
     }
-
-    cpu_trace->activities.insert(
-      cpu_trace->activities.end(),
-      memory_events_.begin(),
-      memory_events_.end());
   }
 
+  uint64_t start_time_;
   std::vector<KinetoEvent> kineto_events_;
   std::unique_ptr<libkineto::CpuTraceBuffer> cpu_trace;
-  std::vector<libkineto::GenericTraceActivity> memory_events_;
```

Comment:
Does this mean that support for memory profiling is removed?

Reply:
no it doesn't, we pass the memory events in kineto_events_ (as we should've)